### ISSUE TITLE
Hotgraphic pin buttons should have ‘aria-haspopup="dialog"' attribute to assist screen readers

### ISSUE THREAD

**Issue by cahirodoherty-learningpool** [adaptlearning/adapt-contrib-hotgraphic#350]
### Subject of the issue/enhancement/features
Interactive elements that display a popup should have an ‘aria-haspopup="dialog"' attribute to assist screen readers.

**Comment by oliverfoster** [comment]
Could you reference a spec / article / another pr that does please?
